bpy_speckle/convert/from_speckle/mesh.py

```diff
@@ -150,11 +147,9 @@ def import_mesh(speckle_mesh, scale=1.0, name=None):
     Convert Mesh object
     """
     if not name:
-        name = speckle_mesh.geometryHash
-        if not name:
-            name = str(speckle_mesh.id)
+        name = speckle_mesh.geometryHash or speckle_mesh.id
 
-    if name in bpy.data.meshes.keys() and False:
+    if "name" in bpy.data.meshes.keys() and False:
```

Comment:
I think ­`"name"` should not be in quotations, as that is an actual variable, not a string...